aidants_connect_web/templates/public_website/politique_confidentialite.html

```diff
@@ -0,0 +1,189 @@
+      <ul class="fr-pl-2w">
+        <li>L'exécution d'une mission d'intérêt public ou relevant de l'exercice de l'autorité publique dont est investi le responsable de traitement au sens de l'article 6-1 e) du RPGD.</li>
+      </ul>
+      <h2 class="fr-mt-4w">Pendant combien de temps conservons-nous ces données&nbsp;?</h2>
+      <div class="fr-table fr-table--bordered fr-table--layout-fixed">
+        <table>
+          <thead>
+            <tr>
+              <th scope="col">Type de données</th>
+              <th scope="col">Durées de conservation</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><strong>Données d'identification de l'usager</strong></td>
+              <td>3 ans en base active après l'expiration de leur dernier mandat et 3 ans supplémentaire dans l'archivage intermédiaire</td>
+            </tr> 
+            <tr>
+              <td><strong>Données d'identification des aidants habilités par les structures</strong></td>
+              <td>3 ans en base active à compter du dernier contact avec l'aidant et 3 ans supplémentaire dans l'archivage intermédiaire</td>
+            </tr> 
+            <tr>
+              <td><strong>Données relatives au mandat</strong></td>
+              <td>3 ans en base active à compter de l'expiration du mandat et 3 ans supplémentaire dans l'archivage intermédiaire</td>
+            </tr> 
+          </tbody>
+        </table>
+      </div>
+      <h2 class="fr-mt-4w">Quels droits avez-vous&nbsp;?</h2>
+      <p class="fr-mb-1w">Vous disposez&nbsp;:</p>
+      <ul class="fr-pl-2w fr-mb-1w">
+        <li>D'un droit d'information et d'un droit d'accès à vos données&nbsp;;</li>
+        <li>D'un droit de rectification&nbsp;;</li>
+        <li>D'un droit d'opposition&nbsp;;</li>
+        <li>D'un droit à la limitation du traitement.</li>
+      </ul>
+      <p>Pour les exercer, contactez-nous à&nbsp;: <a href="{% mailto_href AC_CONTACT_EMAIL %}" class="fr-link" target="_blank">contact@aidantsconnect.beta.gouv.fr</a></p>
+      <p class="fr-mb-1w">Ou par voie postale&nbsp;:</p>
+      <p>
+        <strong>Agence Nationale de la Cohésion des Territoire</strong><br/>
+        <em>À l'attention du délégué à la protection des données (DPD)</em><br/>
+        20 avenue de Ségur<br/>
+        75007 Paris
+      </p>
+      <p>Puisque ce sont des droits personnels, nous ne traiterons votre demande que si nous sommes en mesure de vous identifier. Dans le cas où nous ne parvenons pas à vous identifier, nous pouvons être amenés à vous demander une preuve de votre identité.</p>
+      <p>Pour vous aider dans votre démarche, vous trouverez un modèle de courrier élaboré par la CNIL ici&nbsp;: <a href="https://www.cnil.fr/fr/modele/courrier/exercer-son-droit-dacces" target="_blank" class="fr-link">https://www.cnil.fr/fr/modele/courrier/exercer-son-droit-dacces</a></p>
```

Comment:
```suggestion
      <p>Pour vous aider dans votre démarche, vous trouverez un modèle de courrier élaboré par la CNIL ici&nbsp;: <a href="https://www.cnil.fr/fr/modele/courrier/exercer-son-droit-dacces" target="_blank" rel="noopener noreferer" class="fr-link">https://www.cnil.fr/fr/modele/courrier/exercer-son-droit-dacces</a></p>
```

Pro-tips sécurité : Toujours utiliser `rel="noopener noreferer"` avec un `target="_blank"` : https://techstacker.com/html-noopener-attribute/